python/nav/models/cabling.py

```diff
@@ -28,7 +28,12 @@ class Cabling(models.Model):
     closet's jack number to the end user's room number."""
 
     id = models.AutoField(db_column='cablingid', primary_key=True)
-    room = models.ForeignKey(Room, on_delete=models.CASCADE, db_column='roomid')
+    room = models.ForeignKey(
+        Room,
+        on_delete=models.CASCADE,
+        db_column='roomid',
+        related_name="cablings",
```

Comment:
"cabling" is a mass noun, so it is in fact incorrect to pluralize it as "cablings" (just as the plural of "luggage" is "luggage", not "luggages").

`cabling_set` might actually have been the better option here, to indicate it is indeed the plural one-to-many relation we're talking about.